Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Runs on the default latest node LTS image
+# Build stage
+FROM node:lts AS build
+
+## Create the solid community server directory
+RUN mkdir /community-server
+
+## Set current working directory
+WORKDIR /community-server
+
+## Copy the dockerfile's context's community server files
+COPY . .
+
+## Verify if there are known vulnerabilities in the dependencies
+RUN npm audit --production --audit-level=high
+
+## Install and build the Solid community server
+RUN npm ci
+
+
+# Runtime stage
+FROM node:lts
+
+## Add contact informations for questions about the container
```

Comment:
There are a lot of comments like this that are self-evident for anyone accustomed to reading docker files, and a short google search away for the rest. I personally don't like including comments like that, because I think they just clutter the code.

If you do want to include them still to improve the experience for unfamiliar developers, I would recommended putting more spaces or some other indicator between the build and runtime stages. It's a little hard to tell this is a multi-stage build from first glance, because every line has spacing between it.

Reply:
I added two lines.